Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM bitnami/tomcat:7.0.94 as fineract
+
+USER root
+RUN apt-get update -qq && apt-get install -y git openjdk-8-jdk wget
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/jre
+
+RUN mkdir fineract
+COPY . fineract
+
+WORKDIR fineract
+RUN sed -i 's/localhost/fineractmysql/' ./fineract-provider/build.gradle
```

Comment:
Is this for the mysql url? Why is it needed? Given how "./gradlew clean war" doesn't run any of the SQL tasks, can this be removed? If not, perhaps an inline comment to document what it's needed for? (FYI I am hoping to remove the Flyway related Gradle tasks, see https://issues.apache.org/jira/plugins/servlet/mobile#issue/FINERACT-765)

Reply:
In build.gradle, we expect mysql to be running on localhost (I am glad that this is going away with your changes). In the docker-compose environment, the mysql instance can be referenced using the hostname 'fineractmysql'. So, simply replacing that so Flyway can find the DB server.

Reply:
Right, but you (in this Dockerfile) don't even actually run those Flyway tasks (which are hopefully going to go away some time) - you, correctly only `RUN ./gradlew clean war` - and that doesn't actually require _mysql to be running on localhost_ agreed?